templates/demo-store/app/routes/($locale).account.recover.tsx

```diff
@@ -38,14 +38,17 @@ export const action: ActionFunction = async ({request, context}) => {
   }
 
   try {
-    await context.storefront.mutate(CUSTOMER_RECOVER_MUTATION, {
+    const data = await context.storefront.mutate(CUSTOMER_RECOVER_MUTATION, {
       variables: {email},
     });
 
+    assertApiErrors(data.customerRecover);
+
     return json({resetRequested: true});
   } catch (error: any) {
     return badRequest({
-      formError: 'Something went wrong. Please try again later.',
+      formError:
+        error?.message ?? 'Something went wrong. Please try again later.',
```

Comment:
Is the request error from the API what we want to show up in the UI in production?

Reply:
With this, user will be able to see descriptive error message in place of fallback error message.

Possible error messages:
https://shopify.dev/docs/api/storefront/2023-07/enums/CustomerErrorCode

Reply:
@blittle Updated